platformio.ini

```diff
@@ -31,7 +31,7 @@ build_flags =
     -DSERIAL3_RX_BUFFER_SIZE=256 -DSERIAL3_TX_BUFFER_SIZE=256
 platform = teensy
 framework = arduino
-board = teensy31
+board = teensy36
```

Comment:
I cannot merge with this in it.

Why don't you create a section `[env:host36]` so this problem does not come up?

Reply:
Sorry, I did not change back after testing, but the `[env:host36]` is so easy and cute!
Thanks for this!